zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -27,6 +27,8 @@ public final class AuthorizationCheckBehavior {
 
   public static final String UNAUTHORIZED_ERROR_MESSAGE =
       "Unauthorized to perform operation '%s' on resource '%s'";
+  public static final String UNAUTHORIZED_ERROR_MESSAGE_WITH_RESOURCE =
```

Comment:
❓ Should we create static variables for each elements ? Like UNAUTHORIZED_ROLE_ERROR_MESSAGE, UNAUTHORIZED_GROUP_ERROR_MESSAGE, instead of write every time "group name '%s'" or "role name '%s'"

Reply:
I'm gonna leave it as is. There's already some exceptions to the rule, for example on resource deletion. I'd prefer not having all these constants for this and put the responsibility in the processors. They know what resource id they are checking, so they know what to put here.